src/libs/ReportUtils.ts

```diff
@@ -955,6 +955,14 @@ function isProcessingReport(report: OnyxEntry<Report> | EmptyObject): boolean {
     return report?.stateNum === CONST.REPORT.STATE_NUM.SUBMITTED && report?.statusNum === CONST.REPORT.STATUS_NUM.SUBMITTED;
 }
 
+/**
+ * Returns true if the policy is of `instant` submit type and if the report is still being processed (i.e. submitted)
+ */
+function isInstantSubmittedState(report: OnyxEntry<Report> | EmptyObject): boolean {
+    const policy = allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${report?.policyID}`] ?? null;
+    return isProcessingReport(report) && PolicyUtils.isInstantSubmitEnabled(policy);
```

Comment:
I thinkkk we should also add `isExpenseReport` because this shouldn't apply for bills / invoices